spec/spec_helper.rb

```diff
@@ -13,6 +13,12 @@
   Knapsack::Adapters::RSpecAdapter.bind
 end
 
+# Configures knapsack_pro if not set otherwise on .github/workflows/build.yml
+unless ENV['DISABLE_KNAPSACK_PRO']
+  require 'knapsack_pro'
+  KnapsackPro::Adapters::RSpecAdapter.bind
+end
+
```

Comment:
Hum, good point. Reason I've added here is because we still have plenty of specs using spec_helper.rb.

I've just checked and we use in specs contained in folders like request, controllers, helpers, jobs, lib, mailers, migrations, model, performance, queries, requests, routing, serializers, services, support, validators and view specs.

Basically, everything which does not require capybara/cuprite. I guess this could be a criteria to draw the line, and try to apply knapsack only to the "bottleneck" specs which are always system specs, I guess.

Reply:
I'm not sure if we are understanding each other. The spec_helper is loading the base_helper and that one is already doing the setup. So I think that we don't need to do it twice. The base_helper will do it for all.

Reply:
> spec_helper is loading the base_helper

Right, and not the other way around. Thanks for pointing that out.

Knapsack_pro config was removed from spec_helper 2f8a46b9bf51aad3159df29b800ba07c347f0e8b.